### PR TITLE
Cart 166 edit blocking towing

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/managers-page/managers-controller.js [hunk 1]
```diff
@@ -144,9 +144,11 @@
       })
     }
 
-    vm.block = function() {
-      if (!vm.hoursFrom || !vm.minutesFrom || !vm.hoursTo || !vm.minutesTo) {
-        vm.errorMessage = 'Error: Enter all data!';
+    vm.block = function(form) {
+      vm.invalidDate = !(vm.hoursFrom && vm.minutesFrom && vm.hoursTo && vm.minutesTo);
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/managers-controller.js]: Why do we need it in "vm"?

File: dev/publicDev/managers-page/managers-controller.js [hunk 2]
```diff
@@ -156,26 +158,57 @@
         vm.blockingData.dateTo = vm.dateTo.setMinutes(vm.minutesTo);
 
         rulesDataService.blocking(vm.blockingData, vm.property._id)
-          .then(function() {
+          .then(function(success) {
             getBlocking(vm.property._id);
+            // vm.blockingData = {};
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/managers-controller.js]: What for?

File: dev/publicDev/managers-page/managers-controller.js [hunk 3]
```diff
@@ -156,26 +158,57 @@
...
+        vm.blockingDataArr[blockingIndex].hoursFrom);
+      vm.newBlockingData.from = vm.blockingDataArr[blockingIndex].dateFrom.setMinutes(
+        vm.blockingDataArr[blockingIndex].minutesFrom);
+
+      vm.newBlockingData.to = vm.blockingDataArr[blockingIndex].dateTo.setHours(
+        vm.blockingDataArr[blockingIndex].hoursTo);
+      vm.newBlockingData.to = vm.blockingDataArr[blockingIndex].dateTo.setMinutes(
+        vm.blockingDataArr[blockingIndex].minutesTo);
+
+        rulesDataService.updateBlocking(vm.newBlockingData, id)
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/managers-controller.js]: Something wrong with indentation

File: dev/publicDev/managers-page/style.scss [hunk 4]
```diff
@@ -221,12 +226,33 @@
     display: inline-block;
   }
 
+  .blocking-list-input.edit {
+    width: 160px;
+    margin-left: 5px;
+    border: 1px solid #ccc;
+  }
+
+  select {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/style.scss]: Use class name

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/managers-page/managers-controller.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/managers-page/managers-controller.js]: because if it is in if it doesn't work

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**